app/containers/EditQuestion/index.js

```diff
@@ -90,10 +90,22 @@ const EditQuestion = ({
     [balance],
   );
 
+  const titleMessage = useMemo(
+    () => {
+      if (question?.postType === 0)
+        return translationMessages[locale][messages.title.id[1]];
```

Comment:
Can you change the order of items in `messages.title.id` and simplify expressions to `messages.title.id[postType]`?

If not then use `POST_TYPE` constant instead of hardcoding values.